builder-support/debian/recursor/debian-buster/pdns-recursor.postinst

```diff
@@ -5,6 +5,9 @@ case "$1" in
   configure)
     addgroup --system pdns
     adduser --system --home /var/spool/powerdns --shell /bin/false --ingroup pdns --disabled-password --disabled-login --gecos "PowerDNS" pdns
+    if [ ! -e /etc/powerdns/recursor.conf -a ! -e /etc/powerdns/recursor.yml ]; then
+      cp /etc/powerdns/recursor.yml-dist /etc/powerdns/recursor.yml
```

Comment:
1) this leaves recursor.yml-dist in /etc/powerdns with possible user confusion
2) this doesn't register recursor.yml as a conffile with dpkg (but recursor.yml-dist would be!)

Reply:
Looking at https://manpages.debian.org/testing/dpkg-dev/deb-conffiles.5.en.html I think we're SOL with this approach and instead must install recursor.yml (at build time). 
But that also means migrating old installs!